net/grpc/gateway/docker/prereqs/Dockerfile

```diff
@@ -12,39 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM grpcweb/common
 
-ARG MAKEFLAGS=-j8
-ARG BUILDIFIER_VERSION=1.0.0
-ARG BAZEL_VERSION=4.1.0
+# Using multi-stage buid (See: https://docs.docker.com/develop/develop-images/multistage-build/)
 
-RUN echo "\nloglevel=error\n" >> $HOME/.npmrc
+######################################
+# Stage 1: Fetch binaries
+######################################
+# node:12-stretch depends is based on buildpack-deps:stretch
+FROM buildpack-deps:stretch AS prepare
 
-WORKDIR /github/grpc-web
+ARG BUILDIFIER_VERSION=1.0.0
+ARG PROTOBUF_VERSION=3.17.3
```

Comment:
Perhaps consider protobuf `3.18.0` they have just released.

Reply:
Good idea! But since we need to update the git submodules too maybe doing a follow-up would be better :)